lib/generators/active_admin/assets/templates/tailwind.config.js

```diff
@@ -1,7 +1,9 @@
-const execSync = require('child_process').execSync;
+import { execSync } from 'child_process';
+import plugin from '@activeadmin/activeadmin/plugin';
```

Comment:
Let's give this an explicit name to make it easier to replace the location where the variable will be used which we need for the generated dev and test apps.

```suggestion
import activeAdminPlugin from '@activeadmin/activeadmin/plugin';
```